region/brinstar/blue/Blue Brinstar Boulder Room.json

```diff
@@ -237,6 +239,16 @@
         "This strat is easily bypassed by jumping through the door, or if the door can be opened."
       ]
     },
+    {
+      "link": [2, 1],
+      "name": "Speedy Airball",
+      "requires": [
+        {"doorUnlockedAtNode": 2},
+        "SpeedBooster",
+        "canLateralMidAirMorph",
+        "canCarefulJump"
+      ]
```

Comment:
This should be fine because at this difficulty it can be assumed you can cj+dg. I want to add a spawn at node here sometime soon too.